ci-operator/config/redhat-operator-ecosystem/cvp/redhat-operator-ecosystem-cvp-ocp-4.8.yaml

```diff
@@ -0,0 +1,27 @@
+releases:
+  latest:
+    release:
+      channel: stable
+      version: "4.8"
```

Comment:
4.8 was not released yet so this will fail to resolve

Reply:
Yeah, I figured we wanted to have this config ready in advance so we can switch over from the CVP side as soon as it's GA, but forgot that it's going to fail CI until then :man_facepalming:

Reply:
You can use a nightly or something similar until then though: check out the `candidate` examples from https://docs.ci.openshift.org/docs/architecture/ci-operator/#testing-with-an-existing-openshift-release